### ISSUE TITLE
Awesome project! But I would like to know how he installs the app on WebUI?

### ISSUE THREAD

**Issue by 175Roy** [bytedance/res-adapter#1]
The current operation is too complicated for us newbies, is it possible to integrate it into the WebUI so that everyone can use your technology visually, thanks!

**Comment by jiaxiangc** [comment]
Thank you for your support. We are currently planning to add support for ResAdapter in both ComfyUI and WebUI, making it more convenient for everyone to use. Please stay tuned for our updates.

**Comment by jiaxiangc** [comment]
Before we support resadapter in ComfyUI and WebUI, we provide a minimal [example](https://github.com/bytedance/res-adapter?tab=readme-ov-file#example) for inference with resadapter.

**Comment by jiaxiangc** [comment]
If you want to use ResAdapter in WebUI now, you can just load resolution lora for low resolution inference.

**Comment by 175Roy** [comment]
Thanks for the prompt response, I tried to install the comfyui version, and after installing it all successfully as requested, I couldn't find any "Res" related words in the node, I hope there is a schematic to show in the project description on the home page, I believe most of them are enthusiasts, it's too hard for us, kudos to you for being so efficient!

**Comment by jiaxiangc** [comment]
@175Roy For resolution interpolation, you only need load resolution lora in LoRA Node. It actually is lora.

**Comment by 175Roy** [comment]
> @175Roy For resolution interpolation, you only need load resolution lora in LoRA Node. It actually is lora.

He seems to have a conflict with "Cropface" that is causing my Comfyui to continually report errors, looking forward to your more advanced Comfyui versions, great work, cheers!

**Comment by AMEERAZAM08** [comment]
Demo Out on GPU  @175Roy 
@jiaxiangc  https://huggingface.co/spaces/ameerazam08/Res-Adapter-GPU-Demo

**Comment by jiaxiangc** [comment]
> Demo Out on GPU @175Roy @jiaxiangc https://huggingface.co/spaces/ameerazam08/Res-Adapter-GPU-Demo

We will make some changes for this demo. Now, the results should be improved. Please stay tuned for our updates within three days.

**Comment by jiaxiangc** [comment]
@175Roy 

We support demo in gradio demo in replicate.com by @chenxwh. Here https://replicate.com/bytedance/res-adapter. 

Huggingface demo by @AMEERAZAM08. Here https://huggingface.co/spaces/ameerazam08/Res-Adapter-GPU-Demo

**Comment by AMEERAZAM08** [comment]
Yes I checked that can we connect on LinkedIn if possible, we can explore more .

**Comment by jiaxiangc** [comment]
> Yes I checked that can we connect on LinkedIn if possible, we can explore more .
@AMEERAZAM08 Thanks for your contribution. We can let more people use ResAdapter for better generation.

**Comment by 175Roy** [comment]
> @175Roy
> 
> We support demo in gradio demo in replicate.com by @chenxwh. Here https://replicate.com/bytedance/res-adapter.
> 
> Huggingface demo by @AMEERAZAM08. Here https://huggingface.co/spaces/ameerazam08/Res-Adapter-GPU-Demo

I am very much looking forward to experiencing the 1.5K resolution version of Comfyui. Is there an approximate release time?

**Comment by jiaxiangc** [comment]
We have released the code about ComfyUI-ResAdapter.

**Comment by andupotorac** [comment]
> If you want to use ResAdapter in WebUI now, you can just load resolution lora for low resolution inference.

Hi. I'm looking to run this on A1111, so I went to HF to download the LORA. I want to ask why are there two files there and which one should we use?

https://huggingface.co/jiaxiangc/res-adapter/tree/main/resadapter_v2_sd1.5
